samples/httpbin/headless-httpbin.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
```

Comment:
It would be better to add a E2E test for Headless service.

Reply:
I will add it